danger-kotlin-kts/src/main/kotlin/systems/danger/kts/DangerFileScriptDefinition.kt

```diff
@@ -64,7 +65,7 @@ class DangerFileKtsConfigurator : RefineScriptCompilationConfigurationHandler {
 
     private val resolver = CompoundDependenciesResolver(
         FileSystemDependenciesResolver(DANGER_LIBS_FLAT_DIR),
-        IvyResolver()
+        MavenDependenciesResolver()
```

Comment:
Everything in this file is borrowed from https://github.com/danger/kotlin/pull/199/